crypto/rand/rand_lcl.h

```diff
@@ -17,21 +17,24 @@
 # include <openssl/ec.h>
 # include "internal/rand.h"
 
-/* we require 256 bits of randomness */
-# define RANDOMNESS_NEEDED (256 / 8)
+/* Amount of randomness (in bytes) we want for initial seeding. */
+# define RANDOMNESS_NEEDED              (128 / 8)
```

Comment:
128 bits is secure and reseeding 256 bits every time is wasteful of a system resource.

At some point (when I get to it :) I'd like to see a separation between the first seeding and subsequent reseedings -- things like `getpid` aren't helpful more than once.  In this case I'd probably go for 256 bits for the initial seeding and 128 for later ones.